examples/SLAM-camera/js/index.js

```diff
@@ -23,12 +20,23 @@ import {HitTestLocationXR8} from './hit-test-location-xr8.js';
 import {SpawnMeshOnReticle} from './spawn-mesh-on-reticle.js';
 /* wle:auto-imports:end */
 
+import {loadRuntime} from '@wonderlandengine/api';
+import * as API from '@wonderlandengine/api'; // Deprecated: Backward compatibility.
```

Comment:
ideally remove that if you dont use global WL. We did that just for backward compatibility, but your project is brand new :)